packages/playwright/src/fixture.ts

```diff
@@ -306,16 +313,18 @@ export async function replayFixture(
           category: step.category,
           title: step.title,
           params: step.params || {},
-          // TODO: this is unhelpful as it points to the monkey-patched function itself
           frames: step.location ? [step.location] : [],
```

Comment:
I just pushed out the change to use the whole `filteredStack` here. Could you take another look?

Reply:
> if Playwright passes a data.location on its own then I'd assume it might be a location captured on purpose

TBH I think you're overestimating Playwright here. It sometimes reports locations in our reporter here so I think it's more likely that your `filteredStack` is useful rather than Playwright's `location`.

Reply:
I recently fixed a bunch of things - like even duplicated steps being produced by our fixture. Those problems could impact what you are mentioning so there is a chance that those are already gone. 

> TBH I think you're overestimating Playwright here.

It might be 😅 I'd like to continue with what I have here at the time - we can always revisit and even add concrete tests when we spot issues with this.